src/components/Avatar/AvatarIcon.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+
+const propTypes = {
+    tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+    className: PropTypes.string,
+    link: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+}
+
+const defaultProps = {
+    tag: 'div',
+    anchor: 'a',
+    link: '#'
+}
+
+const AvatarIcon = (props) => {
+    const {className, tag:Tag, anchor:Anchor, size, href, link, type, ...attributes } = props
+    const typeClass = classNames({[`avatar ${type} ${size}`]:true})
```

Comment:
```js
const typeClass = classNames('avatar',{
  [`size-${size}`]: size,
  [`avatar-${color}]: color
})
```

Reply:
@dej611 you want to change `type` to `color`?

Reply:
```javascript
 const typeClass = classNames('avatar',{
        [`size ${size}`]: size,
        [`avatar ${color}`]: color
      })
```
This is the correct code.

Reply:
You need the dash as size classes look like `size-xs`, `size-sm`, etc... => https://italia.github.io/bootstrap-italia/docs/componenti/avatar/

Reply:
Same for `color` => `avatar-primary`, `avatar-secondary`, ...